modules/extensions/src/terrain/terrain-cover.ts

```diff
@@ -211,15 +222,9 @@ function getTile(layer: Layer): Tile2DHeader | null {
   return null;
 }
 
-function intersect(
-  b1: GeoBoundingBox | NonGeoBoundingBox,
-  b2: GeoBoundingBox | NonGeoBoundingBox
-): boolean {
-  if ('west' in b1 && 'west' in b2) {
-    return b1.west < b2.east && b2.west < b1.east && b1.south < b2.north && b2.south < b1.north;
-  }
-  if ('left' in b1 && 'left' in b2) {
-    return b1.left < b2.right && b2.left < b1.right && b1.top < b2.bottom && b2.top < b1.bottom;
+function intersect(b1?: [number[], number[]], b2?: [number[], number[]]): boolean {
```

Comment:
Sounds like we want a bounding-box.ts file that contains the type and the util? It could also have a makeBoundingBox that accepts a bunch of variants...